src/overlays/effects/ovl_Effect_Ss_Bomb/z_eff_ss_bomb.c

```diff
@@ -7,8 +7,10 @@
 #include "z_eff_ss_bomb.h"
 #include "assets/objects/gameplay_keep/gameplay_keep.h"
 
+#define LIFESPAN 20
+
```

Comment:
can probably prefix this, though just "BOMB_ isnt great lmao
maybe EXPLOSION, idk

Reply:
the way it's used makes me think it was meant to be 1 second and this dev just hardcoded it to 20 frames

Reply:
that is the case, but the define allows you to change the duration in one place and have all of the relevant places update at the same time so that it still works correctly

Reply:
oh yeah I meant instead of them using the R_UPDATE_RATE reg or something to get 20

Reply:
im lost on what youre trying to point out then 😅 my comment is just saying to add a prefix to the define name so that it more clearly belongs to this file/system. hardcoded timer durations is nothing special at all

Reply:
Sorry I was really thinking out loud and I'm realizing this comment thread isn't the place for it. I was lamenting at another instance of devs using hardcoded framerate instead of referencing a global framerate value. I agree completely with what you're saying about the define name.

Reply:
-> `EFFSSBOMB_LIFESPAN` :shrug: